src/status_im/components/carousel/styles.cljs

```diff
@@ -5,10 +5,15 @@
 
 (defn content-container [sneak gap]
   {:paddingLeft (+ 0 (quot gap 2))
```

Comment:
The same problem as was on the next line. I know that it was written before, but it so close :)